cmd/callbacks/create.go

```diff
@@ -0,0 +1,104 @@
+package callbacks
+
+import (
+	"context"
+	"encoding/json"
+	"strings"
+	"time"
+
+	"github.com/resonatehq/resonate/pkg/client"
+	v1 "github.com/resonatehq/resonate/pkg/client/v1"
+	"github.com/spf13/cobra"
+)
+
+var createCallbacksExample = `
+# Create a callback
+resonate callback create --promise-id foo --root-promise-id bar --timeout 1h --recv default
+
+# Create a callback with url
+resonate callback create --promise-id foo --root-promise-id bar --timeout 1h --recv poll://default/6fa89b7e-4a56-40e8-ba4e-78864caa3278
+
+# Create a callback with object
+resonate callback create --promise-id foo --root-promise-id bar --timeout 1h --recv {"type": "poll", "data": {"group": "default", "id": "6fa89b7e-4a56-40e8-ba4e-78864caa3278"}}
+`
+
+func CreateCallbackCmd(c client.Client) *cobra.Command {
+	var (
+		promiseId     string
+		rootPromiseId string
+		timeout       time.Duration
+		recv          string
+	)
+	cmd := &cobra.Command{
+		Use:     "create ",
+		Short:   "Create callbacks",
+		Example: createCallbacksExample,
+		RunE: func(cmd *cobra.Command, args []string) error {
+
+			// What should requestID be?
+			paramsByte, err := json.Marshal(struct {
```

Comment:
We can remove requestId from the CLI, its an optional parameter that is more handy for web applications, you can pass nil to CreateCallbackWithResponse for the params.